controllers/infra/drivers/aliyun/create_instance.go

```diff
@@ -0,0 +1,174 @@
+package aliyun
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/labring/sealos/pkg/utils/retry"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
+	v1 "github.com/labring/sealos/controllers/infra/api/v1"
+	"github.com/labring/sealos/controllers/infra/common"
+)
+
+var userData = `#!/bin/bash
```

Comment:
Seems ali cloud support to use root to ssh by default, so no needs this userData.

Reply:
okay, i've removed it

Reply:
> Seems ali cloud support to use root to ssh by default, so no needs this userData.

okay, i've removed it